docs/community/governance.md

```diff
@@ -0,0 +1,118 @@
+# GradleUp Governance Charter
+
+## What is GradleUp?
+
+GradleUp is a volunteer-driven umbrella GitHub organization
+focusing on simplifying developer experience with [Gradle Build Tool (a.k.a. Gradle)](https://gradle.org/),
+in particular for Java, Kotlin and Android projects.
+We host components and ensure their long-term sustainability.
+
+Our scope includes all things that help to achieve great user and developer experience when working with Gradle.
+It includes but not limited to plugins, developer tools, integrations, documentation or community content.
+
+## Formal Side
+
+The GradleUp organization is a volunteer driven community
+operating in a public interest in a non-profit mode and
+providing no commercial services.
+As it is written in the Apache License v2:
+
+> Licensor provides the Work (and each Contributor provides its Contributions) on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied, including, without limitation, any warranties or conditions of TITLE, NON-INFRINGEMENT, MERCHANTABILITY, or FITNESS FOR A PARTICULAR PURPOSE. You are solely responsible for determining the appropriateness of using or redistributing the Work and assume any risks associated with Your exercise of permissions under this License.
+
+Gradle and the Gradlephant logo (the "Gradle Marks") are registered trademarks of [Gradle, Inc.](https://gradle.com/) and/or its subsidiaries.
+In the GradleUp organization, "Gradle" typically means "Gradle Build Tool" and does not reference Gradle Inc. and/or its subsidiaries.
+All references to the trademark represent a fair use,
+and do not imply endorsement.
+
+## Structure
+
+The project is led by a small team of passionate Gradle users and contributors.
+This [GradleUp Team](./team.md) is to be considered as a top governance and steering entity for the project.
+
+The project also includes maintainers and contributors.
+
+## Roles
+
+* **Contributor** - any individual who participates in the community and 
+  adds value to in one form or another,
+  including but not limited to code, documentation ,automation, advocacy or
+  community support.
+* **Team Member** -
+  A GradleUp contributor listed on the [GradleUp Team page](./team.md).
+  They participate in the project steering as whole.
+  New team members are added by the decision of the current team members.
+* **Org Admin** - A team member with administrative access to the GitHub
+  organization and/or other community resources.
+  Permissions and access are granted based on the decision of the current team members.
+* **Maintainer** - A GradleUp contributor who maintains a component hosted on GradleUp.
+  It includes roles _Triage_ or above.
+  Components may explicitly or implicitly introduce more Maintainer sub-roles.
```

Comment:
Suggestion to simplify and have only 3 roles:

- Member
  - access to a specific GitHub repository
- Maintainer
  - admin access to a specific GitHub repository
- Admin
  - admin access to GitHub org, Sonatype, Gradle plugin portal and DNS

Contributors can be anyone, right, we do accept pull requests, etc... Just like any other OSS orgs. I don't expect the to be part of the GradleUp team (unless they want to of course)

Reply:
It excludes the "GradleUp Team member" which is the current steering entity. I would prefer to keep it as is for now, given we may want to have less org admins with permissions.

Reply:
> It excludes the "GradleUp Team member" which is the current steering entity

I'd say everyone in the GradleUp organization would effectively be a team member (https://github.com/orgs/GradleUp/people)? 

* You're a repo maintainer => you're part of the GradleUp org
* You're a repo maintainer and you invite a new collaborator => they're part of the GradleUp org

There might be other ways but it's handy to have https://github.com/orgs/GradleUp/people as the source of truth for the "GradleUp" team.